integration-tests/cli/csv-parser-order.it.js

```diff
@@ -0,0 +1,183 @@
+import { exec } from 'child_process'
+import fs from 'fs'
+import path from 'path'
+import streamtest from 'streamtest'
+import tmp from 'tmp'
+import {
+  AddReturnInfoCsvParser,
+  LineItemStateCsvParser,
+} from '../../packages/csv-parser-order/src/index'
+import { version } from '../../packages/csv-parser-order/package.json'
+
+describe('CSV and CLI Tests', () => {
+  const binPath = './integration-tests/node_modules/.bin/csvparserorder'
+  const samplesFolder = './packages/csv-parser-order/test/data/'
+
+  describe('CLI basic functionality', () => {
+    test('should print usage information given the help flag', (done) => {
+      exec(`${binPath} --help`, (error, stdout, stderr) => {
+        expect(String(stdout)).toMatch(/help/)
+        expect(error && stderr).toBeFalsy()
+        done()
+      })
+    })
+
+    test('should print the module version given the version flag', (done) => {
+      exec(`${binPath} --version`, (error, stdout, stderr) => {
+        expect(stdout).toBe(`${version}\n`)
+        expect(error && stderr).toBeFalsy()
+        done()
+      })
+    })
+
+    test('should write output to file', (done) => {
+      const csvFilePath = `${samplesFolder}return-info-sample.csv`
+      const jsonFilePath = tmp.fileSync().name
+
+      exec(`${binPath} -i ${csvFilePath} -o ${jsonFilePath} -t returninfo`,
+        (cliError, stdout, stderr) => {
+          expect(cliError && stderr).toBeFalsy()
+
+          fs.readFile(jsonFilePath, { encoding: 'utf8' }, (error, data) => {
+            expect(data.match(/returnInfo/)).toBeTruthy()
```

Comment:
I find this assertion not to be very helpful, allowed several bugs to pass through during the `csv-parser-price` migration.

Let's assert the exact data expected, not sure a regex...